drizzlepac/hlautils/hla_flag_filter.py

```diff
@@ -573,7 +561,7 @@ def hla_saturation_flags(drizzled_image, flt_list, catalog_name, catalog_data, p
         phot_table_rows = catalog_data
         for i, table_row in enumerate(phot_table_rows):
             if saturation_flag[i]:
-                table_row[column_titles["flag_coltitle"]] = int(table_row[column_titles["flag_coltitle"]]) | 4
+                table_row["Flags"] = int(table_row["Flags"]) | 4
```

Comment:
What are these hard-coded values (1, 4, 8, 16)?  They should be in a configuration file.  If these values will never change, perhaps they could be an enumerated list or a dictionary which describes their purpose.  Since we are trying to get all the pieces of our code merged so we can implement our last bits of cleanup, addressing this can wait until the next iteration of cleanup.